addons/rating/controllers/main.py

```diff
@@ -18,22 +16,23 @@ def open_rating(self, token, rate, **kwargs):
         if not rating:
             return request.not_found()
         rate_names = {
+            10: _("satisfied"),
             5: _("not satisfied"),
-            1: _("highly dissatisfied"),
-            10: _("satisfied")
+            1: _("highly dissatisfied")
         }
         rating.write({'rating': rate, 'consumed': True})
         lang = rating.partner_id.lang or get_lang(request.env).code
         return request.env['ir.ui.view'].with_context(lang=lang).render_template('rating.rating_external_page_submit', {
             'rating': rating, 'token': token,
-            'rate_name': rate_names[rate], 'rate': rate
+            'rate_names': rate_names, 'rate': rate
         })
 
-    @http.route(['/rating/<string:token>/<int:rate>/submit_feedback'], type="http", auth="public", methods=['post'], website=True)
-    def submit_rating(self, token, rate, **kwargs):
+    @http.route(['/rating/<string:token>/submit_feedback'], type="http", auth="public", methods=['post'], website=True)
+    def submit_rating(self, token, **kwargs):
         rating = request.env['rating.rating'].sudo().search([('access_token', '=', token)])
         if not rating:
             return request.not_found()
+        rate = kwargs.get('rate')
```

Comment:
As I have updated code with 'assert' statement to check proper 'rate' but it will raise an error page with 'incorrect rating' message.  For better user incorrect rating response, we have to handle that error with proper template.